changes/next/sieve_mdn_original_recipient_header

```diff
@@ -0,0 +1,21 @@
+Description:
+
+The new `sieve_mdn_original_recipient_header` allows an install to get the
+original recipient from a header other than Original-Recipient.
+
+
+Config changes:
+
+This adds `sieve_mdn_original_recipient_header`, a yes/no option which, if set,
```

Comment:
This "yes/no option" is incorrect, since you're providing a header name

Reply:
Fixed in amendment.